monitor/api/alerts.py

```diff
@@ -59,35 +59,42 @@ def _query_alert_type(self, run_operation_args: dict, alert_factory_func: Callab
             logger.error('Failed to parse some alerts.')
             self.success = False
         return AlertsQueryResult(alerts, malformed_alerts)
-    
+
     @classmethod
     def _normalize_alert(cls, alert: dict) -> dict:
         try:
             normalized_alert = copy.deepcopy(alert)
-            test_meta = try_load_json(normalized_alert.get('test_meta'))
-            test_meta = test_meta if test_meta else {}
-            model_meta = try_load_json(normalized_alert.get('model_meta'))
-            model_meta = model_meta if model_meta else {}
+            test_meta = try_load_json(normalized_alert.get('test_meta')) or {}
+            model_meta = try_load_json(normalized_alert.get('model_meta')) or {}
+            snapshot_meta = try_load_json(normalized_alert.get('snapshot_meta')) or {}
 
             subscribers = []
-            direct_subscribers = test_meta.get('subscribers', [])
+            test_subscribers = test_meta.get('subscribers', [])
             model_subscribers = model_meta.get('subscribers', [])
```

Comment:
I think we should have only meta and test_meta instead of having model_meta and snapshot_meta and test_meta as you won't have model_meta and snapshot_meta together, it's either you have model_meta or snapshot_meta and therefore there should be only 'meta' that contains either one of them (also explained below).